key.go

```diff
@@ -176,15 +176,40 @@ const (
 )
 
 // KeyModifier represents any modifier key (shift etc.) that is being pressed together with a key.
+//
+// Since: 2.2
 type KeyModifier int
 
 const (
 	// KeyModifierShift represents a shift key being held
+	//
+	// Since: 2.2
 	KeyModifierShift KeyModifier = 1 << iota
 	// KeyModifierControl represents the ctrl key being held
+	//
+	// Since: 2.2
 	KeyModifierControl
 	// KeyModifierAlt represents either alt keys being held
+	//
+	// Since: 2.2
 	KeyModifierAlt
 	// KeyModifierSuper represents either super keys being held
+	//
+	// Since: 2.2
 	KeyModifierSuper
 )
+
+const (
+	// RuneModifierAlt is the rune that represents the Alt key.
+	//
+	// Since: 2.2
+	RuneModifierAlt = '⌥'
```

Comment:
Do these need to be public? Isn’t it a rendering detail rather than part of the user API?

Reply:
Yes, you’re right.